xbmc/addons/PVRClient.cpp

```diff
@@ -808,6 +808,17 @@ PVR_ERROR CPVRClient::IsPlayable(const CConstPVREpgInfoTagPtr &tag, bool &bIsPla
   return retVal;
 }
 
+void CPVRClient::WriteFileItemProperties(const PVR_NAMED_VALUE *properties, unsigned int iPropertyCount, CFileItem &fileItem)
```

Comment:
Minor but `Set` fits better as `Write`

Reply:
And put fileitem param as first?

Reply:
Not really, as all CPVRClient methods copying data back and forth between add-on and Kodi are prefixed "Write". I actually started with "Set" and then changed my code  to fit  the wording already used here.

Reply:
Ok don't know that

Reply:
Re second comment: no, as I ffollow source target order.